spec/systems/keywords/list_keywords_spec.rb

```diff
@@ -13,19 +15,34 @@
   end
 
   context 'given a list of keywords' do
-    it 'displays a link to keywords#show for each keyword and displays the pagination nav' do
-      Fabricate.times(2, :keyword)
+    it 'displays a link to keywords#show for succeeded keyword and displays the pagination nav' do
+      pending_keyword = Fabricate(:keyword)
+      succeeded_keyword = Fabricate(:parsed_keyword)
 
       visit(root_path)
 
       within('tbody') do |tbody|
-        tbody.all('tr').each do |row|
-          keyword_id = row['data-keyword-id']
-          expect(row).to(have_link('', href: keyword_path(keyword_id)))
-        end
+        pending_row = tbody.find("tr[data-keyword-id=\"#{pending_keyword.id}\"]")
+        expect(pending_row).not_to(have_link('', href: keyword_path(pending_keyword.id)))
+
+        succeeded_row = tbody.find("tr[data-keyword-id=\"#{succeeded_keyword.id}\"]")
+        expect(succeeded_row).to(have_link('', href: keyword_path(succeeded_keyword.id)))
       end
 
       expect(page).to(have_selector('.keywords-table__pagination'))
     end
+
+    it 'updates the UI after the keyword finish processing', vcr: 'google/google_no_ads' do
+      keyword = Fabricate(:keyword)
+
+      visit(root_path)
+      perform_enqueued_jobs do
+        ScrapeKeywordJob.perform_later(keyword_id: keyword.id)
+      end
+
+      within("tbody > tr[data-keyword-id=\"#{keyword.id}\"]") do |tr|
+        expect(tr.find('td[data-testid="status"]')).to(have_content('Succeeded'))
```

Comment:
Let's localize the `Succeeded`

Reply:
Fixed in 9cdf12a0d27bd886e7cc3bd5248a02835b4523b0